ee/hogai/tools/create_insight.py

```diff
@@ -512,6 +513,7 @@
 
 
 class CreateInsightToolArgs(BaseModel):
+    title: str = Field(description="A short title for the insight.")
     query_description: str = Field(description="A plan of the query to generate based on the template.")
```

Comment:
Swapping these fields might improve generations a bit since the title would be a summary of the plan then:

```suggestion
    query_description: str = Field(description="A plan of the query to generate based on the template.")
    title: str = Field(description="A short title for the insight.")
```